### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by regebro**
No real code changes at all, making a pull request to have somebody else to a code review anyway.

### CODE REVIEW

File: src/z3c/currency/converter.py [hunk 1]
```diff
@@ -20,10 +20,10 @@
 from z3c.currency import interfaces
 from zope.i18n import format
 
+@zope.interface.implementer(IDataConverter)
 class CurrencyConverter(object):
     """Converts currency fields to text representations."""
     zope.component.adapts(interfaces.ICurrency, IWidget)
-    zope.interface.implements(IDataConverter)
```
Comment (mgedmin) [inline src/z3c/currency/converter.py:26]: Shouldn't `adapts()` be also moved to a class decorator `@adapter(...)`?

**Comment by mgedmin** [comment]
I would like to see diffs for CHANGES.rst, tox.ini and .travis.yml before this is merged.

File: src/z3c/currency/converter.py (hunk 1 above)
Comment (regebro) [inline src/z3c/currency/converter.py:26]: Yes, absolutely. It seems like I never got around to writing fixers for that. My bad. Will fix manually for now.

**Comment by mgedmin** [comment]
Thank you!

**Pull request merged**